applications/DemStructuresCouplingApplication/python_scripts/sp_statistics.py

```diff
@@ -0,0 +1,144 @@
+import math
+import matplotlib.pyplot as plt
+import numpy as np
+import h5py
```

Comment:
not sure if applicable; have you considered using the HDF5-Application?

Reply:
Thanks, I know that we had the possibility of using it but, at least for what we needed, I think this implementation is simple and it's enough.